runtime/main/main.cpp

```diff
@@ -56,15 +60,33 @@ void setupUartDriver() {
         .parity    = UART_PARITY_DISABLE,
         .stop_bits = UART_STOP_BITS_1,
         .flow_ctrl = UART_HW_FLOWCTRL_DISABLE,
-        .rx_flow_ctrl_thresh = 127,
+        .rx_flow_ctrl_thresh = 63,
         .source_clk = UART_SCLK_APB,
     };
-    uart_driver_install( UART_NUM_0, 4096, 0, 0, NULL, 0 );
+    uart_driver_install( UART_NUM_0, 4096, 0, 0, NULL, ESP_INTR_FLAG_IRAM );
```

Comment:
Jen pro zaznamenání; ve čtvrtek jsem se bavili, že pokud máme problém s průtokem, zkusil bych se na driver kompletně vykašlat a přímo v našem kódu číst UARTová FIFA. Stálo by za to vyzkoušet.